lmos-runtime-core/src/main/kotlin/org/eclipse/lmos/runtime/core/inbound/ConversationHandler.kt

```diff
@@ -104,54 +105,12 @@ class DefaultConversationHandler(
                     agentName,
                     agentAddress,
                     routingInformation.subset,
-                ).onEach {
-                    log.info("Agent Response: ${'$'}{it.content}")
-                }
+                )
         }
 
-    /**
-     * Retrieves a RoutingInformation for a conversation.
-     * <p>
-     * First, this method attempts to load the routing information from the cache.
-     * If no cached data is available, it fetches the routing information from the {@code AgentRegistryService},
-     * stores it in the cache, and then returns it.
-     *
-     * @param tenantId        the tenant ID
-     * @param conversationId  the conversation ID
-     * @param subset          optional subset parameter for agent selection
-     * @param conversation    the current conversation
-     * @return the routing information for the conversation
-     */
-    private suspend fun retrieveRoutingInformation(
-        tenantId: String,
-        conversationId: String,
-        subset: String?,
-        conversation: Conversation,
-    ): RoutingInformation {
-        // Lookup cached RoutingInformation
-        val cachedRoutingInformation = lmosRuntimeTenantAwareCache.get(tenantId, ROUTES, conversationId)
-        // Check if cached RoutingInformation can be reused and subset has not changed
-        val routingInformation =
-            if (cachedRoutingInformation != null) {
-                cachedRoutingInformation
-            } else {
-                val fetched =
-                    agentRegistryService.getRoutingInformation(
-                        tenantId,
-                        conversation.systemContext.channelId,
-                        subset,
-                    )
-                log.debug("Caching routing information: {}", fetched)
-                lmosRuntimeTenantAwareCache.save(
-                    tenantId,
-                    ROUTES,
-                    conversationId,
-                    fetched,
-                    lmosRuntimeConfig.cache.ttl,
```

Comment:
Please remove the old TTL configuration from all application YAML files and config maps in the service and GraphQL modules.